plugins-bundled/internal/input-datasource/jest.config.js

```diff
@@ -1,8 +1,18 @@
-// This file is needed because it is used by vscode and other tools that
-// call `jest` directly.  However, unless you are doing anything special
-// do not edit this file
-
-const standard = require('@grafana/toolkit/src/config/jest.plugin.config');
-
-// This process will use the same config that `yarn test` is using
-module.exports = standard.jestConfig();
+module.exports = {
+  testEnvironment: 'jest-environment-jsdom',
+  preset: 'ts-jest',
+  extensionsToTreatAsEsm: ['.ts'],
+  transform: {
+    '^.+\\.(t|j)sx?$': [
+      'ts-jest',
+      {
+        useESM: true,
+        isolatedModules: true,
+        allowJs: true,
+      },
+    ],
+  },
+  moduleNameMapper: {
+    '^d3-interpolate$': '<rootDir>/__mocks__/d3-interpolate.ts',
```

Comment:
This fixed a problem with jest and esm that I could not fix with transformIgnorepattern

Reply:
Indeed. I spent a long time trying to make it work because we also use it in create-plugin but it didn't work. I suspect something about versions or dependencies or other odd setup I could not decipher.